### ISSUE TITLE
End of Race buttons (Restart, Continue, etc.) are confusing

### ISSUE THREAD

**Issue by kryptot7** [supertuxkart/stk-code#2978]
In the End of Race menu, the buttons because slightly lighter when they're selected. But because there are only two buttons, it's hard to tell if the darker button is the one you're selecting or the lighter button.
Also, if the mouse is on one of the buttons and you focus on the OTHER one with the joystick, it becomes even more confusing.

**Comment by auriamg** [comment]
Agreed, though I would classify this more generally as a need to clarify focus in our UI skins

**Comment by qwertychouskie** [comment]
Maybe some sort of animation on the selected button, e.g. https://jacobspctuneup.tk/STK/newgui/buttondemo.html

**Comment by CodingJellyfish** [comment]
Why the issue is still here? Those buttons are icon buttons now.

**Comment by CodingJellyfish** [comment]
Close this since we use icon buttons now.
